examples/phosh/README.md

```diff
@@ -0,0 +1,6 @@
+## Building
+
+The stage-2 needs to be built natively on the target architecture (armv7 on
+armv7, aarch64 on aarch64).
+
+(Though the tooling will try to build it through cross-compilation!)
```

Comment:
i have set this for cc on my system:

```
  # allow build for aarch64 architecture through emulation
  boot.binfmt.emulatedSystems = [ "aarch64-linux" ];
```

it that needed, recommended or not needed here?

if at least recommended, i think we should link to instructions on how to set it up. https://nixos.wiki/wiki/NixOS_on_ARM#Compiling_through_binfmt_QEMU